### PR TITLE
magento/devdocs#: Layout instructions. Improvement - adding changes to Configure services as web APIs topic

### PR DESCRIPTION
**Pull request by DenisSaltanahmedov**
## Purpose of this pull request

This pull request (PR) adds small changes to the Configure services as web APIs topic
-  Changed file path.
The default file is `app/code/Magento/Webapi/etc/webapi.xsd` or `vendor/magento/module-webapi/etc/webapi.xsd`, in the case the "Vendor" in the path can be confused.
-  Added the missed item to the list of subtopics

## Affected DevDocs pages
-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/service-contracts/service-to-web-service.html

## Links to Magento source code
-  https://github.com/magento/devdocs/blob/master/src/guides/v2.3/extension-dev-guide/service-contracts/service-to-web-service.md

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/service-contracts/service-to-web-service.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ functional_areas:
 
 You can configure a Magento or third-party service as a web [API](https://glossary.magento.com/api).
 
-To [configure a web API](#configure-webapi), you define [XML](https://glossary.magento.com/xml) elements and attributes in the `webapi.xml` XML configuration file for the [module](https://glossary.magento.com/module) for the service. The `webapi.xml` file for your module specifies an XML schema file for validation. By default, this file is `app/code/<VENDOR>/Webapi/etc/webapi.xsd`.
+To [configure a web API](#configure-webapi), you define [XML](https://glossary.magento.com/xml) elements and attributes in the `webapi.xml` XML configuration file for the [module](https://glossary.magento.com/module) for the service. The `webapi.xml` file for your module specifies an XML schema file for validation. By default, this file is `app/code/Magento/Webapi/etc/webapi.xsd` or `vendor/magento/module-webapi/etc/webapi.xsd`.
```
Comment (rogyar) [inline src/guides/v2.3/extension-dev-guide/service-contracts/service-to-web-service.md]: I would suggest adjusting this part to clarify all statements

```suggestion
To [configure a web API](#configure-webapi), you define [XML](https://glossary.magento.com/xml) elements and attributes in the `webapi.xml` XML configuration file for the [module](https://glossary.magento.com/module) for the service. The `etc/webapi.xsd` file for your module specifies an XML schema file for validation. The default XML schema validation rules are stored in `app/code/Magento/Webapi/etc/webapi.xsd` or `vendor/magento/module-webapi/etc/webapi.xsd` file.
```

Thank you

**Comment by DenisSaltanahmedov** [comment]
Hi @rogyar, thanks for your review.
I have fixed the recommendations.

**Comment by dobooth** [comment]
running tests

**Pull request merged**